packages/notebook-extension/src/index.ts

```diff
@@ -38,6 +38,9 @@ import { Widget } from '@lumino/widgets';
 
 import { TrustedComponent } from './trusted';
 
+import { PageConfig, URLExt } from '@jupyterlab/coreutils';
```

Comment:
These imports can be grouped with the other imports above.

Reply:
Got it will update the PR.